site/pages/docs/ref/cal-events/cal-events-fr.hbs

```diff
@@ -29,6 +29,14 @@
 	</ul>
 </section>
 
+<section>
+	<h2>Évaluation et rapport</h2>
+	<ul>
+		<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/cal-events/reports/a11y-1-en.html">Rapport Anglais</a></li>
+		<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/cal-events/reports/a11y-1-fr.html">Rapport Français</a></li>
```

Comment:
```suggestion
		<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A &amp; AA)- <a href="../../../demos/cal-events/reports/a11y-1-en.html">Rapport Anglais</a></li>
		<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A &amp; AA)- <a href="../../../demos/cal-events/reports/a11y-1-fr.html">Rapport Français</a></li>
```

Reply:
Fixed in commit "Updated assessment according to new methodologies"